src/app.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+
+import trainingRoutes from './routes/training';
+
+const app = express();
+const port = 8080;
```

Comment:
Port should comes from config file. You can use node-config library.

Reply:
.